### DISCUSSION TITLE
Disable backup on IOS

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#11938] [Q&A]
Hello.
Is there a way in IOS to disable the i-cloud backup of my app?
I don´t want IOS to backup the sqlite database and some other files.
In addition: can I disable the backup of the secure storage as well?

In Android this is a simple setting. Is there something similar in IOS?

Thanks for your help!

**Comment by Larhei** [comment]
Without testing I would give 
`NSFileManager.SetSkipBackupAttribute (filename, true); // backup will be skipped for this file` 
shot
See [here](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/ios/app-fundamentals/file-system.md) under 
Complying with iOS 5 iCloud backup restrictions for more info
